interactions:
- request:
    body: !!python/unicode '{"direction": "both", "labels": ["_acquisition", "_imageFiles"],
      "rootRecordId": "1010010011AD6E00", "maxdepth": 2}'
    headers:
      Accept: ['*/*']
      Accept-Encoding: ['gzip, deflate']
      Connection: [keep-alive]
      Content-Length: ['115']
      Content-Type: [application/json]
      User-Agent: [python-requests/2.12.1]
    method: POST
    uri: https://geobigdata.io/catalog/v2/traverse
  response:
    body: {string: !!python/unicode '{"message":"''eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJodHRwczovL2RpZ2l0YWxnbG9iZS1wbGF0Zm9ybS5hdXRoMC5jb20vIiwic3ViIjoiYXV0aDB8Z2JkeHw4IiwiYXVkIjoidmhhTkVKeW1MNG0xVUNvNFRxWG11S3RrbjlKQ1lEa1QiLCJleHAiOjE0OTA2NDE4MzMsImlhdCI6MTQ5MDAzNzAzM30.YIFp319yWP2pN4DqW8rFxt5ZPk5l5O9bOWAcidjDD3w''
        not a valid key=value pair (missing equal-sign) in Authorization header: ''Bearer
        eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJodHRwczovL2RpZ2l0YWxnbG9iZS1wbGF0Zm9ybS5hdXRoMC5jb20vIiwic3ViIjoiYXV0aDB8Z2JkeHw4IiwiYXVkIjoidmhhTkVKeW1MNG0xVUNvNFRxWG11S3RrbjlKQ1lEa1QiLCJleHAiOjE0OTA2NDE4MzMsImlhdCI6MTQ5MDAzNzAzM30.YIFp319yWP2pN4DqW8rFxt5ZPk5l5O9bOWAcidjDD3w''."}'}
    headers:
      access-control-allow-origin: ['*']
      connection: [keep-alive]
      content-length: ['652']
      content-type: [application/json]
      date: ['Mon, 27 Mar 2017 18:35:08 GMT']
      via: [kong/0.5.2]
      x-amz-cf-id: [Sx5vHZ-5nE1c0_NuIiLhAGRbRAcj1q7t5nJayn6XwjZSPk0ygakF1g==]
      x-amzn-errortype: [IncompleteSignatureException]
      x-amzn-requestid: [1a71d86a-131c-11e7-af6f-738c20fa821a]
      x-cache: [Error from cloudfront]
    status: {code: 403, message: Forbidden}
version: 1
